scripts/skills/perks/perk_rf_calculated_strikes.nut

```diff
@@ -0,0 +1,51 @@
+this.perk_rf_calculated_strikes <- this.inherit("scripts/skills/skill", {
+	m = {
+		DamageMult = 1.2,
+		InitiativeMult = 0.8
+	},
+	function create()
+	{
+		this.m.ID = "perk.rf_calculated_strikes";
+		this.m.Name = ::Const.Strings.PerkName.RF_CalculatedStrikes;
+		this.m.Description = ::Const.Strings.PerkDescription.RF_CalculatedStrikes;
+		// this.m.Icon = "ui/perks/rf_calculated_strikes.png";	// TODO: add this icon
+		this.m.Type = ::Const.SkillType.Perk;
+		this.m.Order = ::Const.SkillOrder.Perk;
+		this.m.IsActive = false;
+		this.m.IsStacking = false;
+		this.m.IsHidden = false;
+	}
+
+	function onUpdate( _properties )
+	{
+		_properties.InitiativeMult *= this.m.InitiativeMult;
+	}
+
+	function onAnySkillUsed( _skill, _targetEntity, _properties )
+	{
+		if (_skill != this) return;
+
+		if (this.isEnabledFor(_targetEntity))
+		{
+			_properties.DamageTotalMult *= this.m.DamageMult;
+		}
+	}
+
+	function onGetHitFactors( _skill, _targetTile, _tooltip )
+	{
+		if (_targetTile.getEntity() != null && this.isEnabledFor(_targetTile.getEntity()))
```

Comment:
I just realized that here also we should start being safe and use `_targetTile.IsOccupiedByActor` instead of checking for entity being null. Because what if a skill is targeting some non-actor entity e.g. tree or obstacle, then this might crash when it checks for `isTurnStarted()` etc. So:
```squirrel
if (_targetTile.IsOccupiedByActor && this.isEnabledFor(_targetTile.getEntity()))
```